staging/src/k8s.io/client-go/kubernetes/typed/core/v1/fake/fake_pod_expansion.go

```diff
@@ -58,11 +58,12 @@ func (c *FakePods) GetLogs(name string, opts *v1.PodLogOptions) *restclient.Requ
 }
 
 func (c *FakePods) Evict(eviction *policy.Eviction) error {
-	action := core.CreateActionImpl{}
-	action.Verb = "create"
+	action := core.GenericActionImpl{}
```

Comment:
Why is CreateActionImpl wrong? I agree with liggitt's comment in the issue: https://github.com/kubernetes/kubernetes/issues/71598#issuecomment-443227156.